packages/ui/atoms/Icon/IconImg.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Icon Image component
+ * Use `<img>` instead of injecting the SVG file.
+ *
+ * @param string $name The file name of the icon.
+ * @param array $attr Custom attributes.
+ * @param array $css_vars Custom CSS variables to pass to the SVG.
+ * @param string $current_color Custom color for the `currentColor` CSS variable in the SVG.
+ */
+#}
+{%- set svg -%}
+  {{ source('@svg/%s.svg'|format(name), ignore_missing = true) }}
+{%- endset -%}
+
+{% if current_color is defined or css_vars is defined %}
+  {%- set style -%}
+    <style>
+      :root {
+        {% if current_color is defined %}
+          color: {{ current_color }};
+        {% endif %}
+        {% if css_vars is defined %}
+          {% for var, value in css_vars %}
+            --{{ var }}: {{ value }};
+          {% endfor %}
+        {% endif %}
+      }
+    </style>
+  {%- endset -%}
+
+  {% if not 'xmlns="http://www.w3.org/2000/svg"' in svg %}
+    {% set svg = svg|replace({ '<svg': '<svg %s'|format('xmlns="http://www.w3.org/2000/svg"') }) %}
+  {% endif %}
```

Comment:
**issue:** These lines should be placed outside the `current_color` and `css_vars` condition